### PR TITLE
Added 'only' method, updated README, added test

### LINKED ISSUE #40: Suggestion/feature this.form.only(...keys)
**Feature description:**

`this.form.only(...keys)`

How would it work?

Similar to how Laravel's `request()->only()` works.

I saw in the source of this package that there's already a `this.form.data()` method.

Basically it's just that but you can pass it an array of fields that it should return.

___

**Example usage:**

I have a use case where I need to `$emit` only the certain fields from large forms to another Vue component.

A method like this would greatly simplify the process.

```js
const fields = {
    foo: this.form.foo,
    bar: this.form.bar
};

// example usage of only()
// const fields = this.form.only(['foo', 'bar']);

this.form.post('/endpoint')
    .then((response) => window.eventBus.$emit('event', fields))
    .catch((error) => console.log(error.response));
```

### PR DESCRIPTION
**Pull request by raniesantos**
close #40

### CODE REVIEW

**Comment by freekmurze** [comment]
Could you add a test that covers the new function?

**Comment by raniesantos** [comment]
@freekmurze I added a test just now.

**Comment by raniesantos** [comment]
I already added a test, did you guys forget about this PR?

**Comment by raniesantos** [comment]
I already added a test, did you guys forget about this PR?

File: src/Form.js [hunk 1]
```diff
@@ -85,6 +85,22 @@ class Form {
...
+    /**
+     * Fetch specific data for the form.
+     *
+     * @param {array} fields
+     * @return {object}
+     */
+    only(fields) {
+        const data = {};
+
+        fields.forEach(property => {
```
Comment (freekmurze) [inline src/Form.js]: can this be implemented using the `filter` function? 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Then we can get rid of the temporary `data` variable.

Comment (raniesantos) [reply src/Form.js]: I used `reduce` instead so that it still returns an object, `filter` returns an array. See latest commit.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**